class.two-factor-core.php

```diff
@@ -492,6 +497,29 @@ public static function login_form_validate_2fa() {
 
 		wp_set_auth_cookie( $user->ID, $rememberme );
 
+		// Must be global because that's how login_header() uses it.
+		global $interim_login;
+		$interim_login = isset($_REQUEST['interim-login']);
```

Comment:
I'll fix this. Again, it was pulled straight from core (and apparently was missed when added because it obviously doesn't meet their coding standards).